lms/static/scripts/frontend_apps/components/SubmitGradeForm.tsx

```diff
@@ -106,6 +112,9 @@ export default function SubmitGradeForm({ student }: SubmitGradeFormProps) {
         });
         setGradeSaved(true);
       } catch (e) {
+        if (isLTILaunchServerError(e)) {
```

Comment:
`isLTILaunchServerError` is not a great name now, it handles file errors, groups, sections and launch errors. It does that while on the launch app.